tests/cli.rs

```diff
@@ -74,3 +74,25 @@ fn basic_head() -> Result<(), Box<dyn std::error::Error>> {
 
     Ok(())
 }
+
+#[test]
+fn basic_options() -> Result<(), Box<dyn std::error::Error>> {
+    let mut cmd = Command::cargo_bin("ht")?;
+    cmd.arg("-v")
+        .arg("--offline")
+        .arg("--ignore-stdin")
+        .arg("--pretty=format")
+        .arg("options")
+        .arg("httpbin.org");
```

Comment:
Can we change the URL `httpbin.org` to `httpbin.org/json` since the latter has more interesting headers?

```
$ http options
HTTP/1.1 200 OK
access-control-allow-credentials: true
access-control-allow-methods: GET, POST, PUT, DELETE, PATCH, OPTIONS
access-control-allow-origin: *
access-control-max-age: 3600
allow: GET, HEAD, OPTIONS
connection: keep-alive
content-length: 0
content-type: text/html; charset=utf-8
date: Sun, 07 Feb 2021 11:28:45 GMT
server: gunicorn/19.9.0
```

Reply:
Sure, but in that case, we won't be able to test with the _--offline_ flag, as these headers are the ones sent back by the server. Or maybe we should launch a dedicated http server during the tests to send our tests against, instead of verifying the requests sent ? For the time being I'll remove the offline flag and test directly againt httpbin.org.

Reply:
Oh, right. It slipped my mind that we were using the _--offline_ flag for other tests.

> maybe we should launch a dedicated HTTP server during the tests to send our tests against, instead of verifying the requests sent?

Sounds good. Personally, I've been using a small Deno script for my local testing. But it would be nice if there was an automated solution that works with `cargo test`.